.eslintrc.json

```diff
@@ -74,45 +74,51 @@
     "mapHeight": "readonly",
     "scavengerPlayer": "readonly",
     "isMultiplayer": "readonly",
+    "challenge": "readonly",
     "isReceivingAllEvents": "readonly",
```

Comment:
i couldn't find `isReceivingAllEvents` anywhere in the code. remove?

Reply:
Might be a leftover from QtScript days. isReceivingAllEvents is only internally influenced by receiveAllEvents() now.

Reply:
removed https://github.com/Warzone2100/warzone2100/pull/2342/commits/f1e8668ab7de7caec9f57a2f24fa8235da5d8e60